src/test_l3_dsHdmiIn.c

```diff
@@ -0,0 +1,1429 @@
+    ret = dsHdmiInRegisterAviContentTypeChangeCB(hdmiInAviContentTypeChangeCB);
+    if (ret != dsERR_NONE) {
+        UT_LOG_ERROR("Failed dsHdmiInRegisterAviContentTypeChangeCB(cbFun:[0x%0X]) dsError_t:[%s]",
+                        hdmiInAviContentTypeChangeCB, ut_control_GetMapString(dsError_mapTable, ret));
+    }
+    else {
+        UT_LOG_INFO("Result dsHdmiInRegisterAviContentTypeChangeCB(cbFun:[0x%0X]) dsError_t:[%s]",
+                        hdmiInAviContentTypeChangeCB, ut_control_GetMapString(dsError_mapTable, ret));
+    }
+
+    UT_LOG_INFO("Out %s", __FUNCTION__);
+}
+
+/**
+* @brief This test get number of input ports.
+*
+* This test function gets HdmiInput ports on platform.
+*
+* **Test Group ID:** 03@n
+* **Test Case ID:** 002@n
+*
+* **Test Procedure:**
+* Refer to Test specification documentation
+* [ds-hdmi-in_halSpec.md](../../docs/pages/ds-hdmi-in_halSpec.md)
+*/
+void test_l3_HdmiIn_get_inputports(void)
+{
+    gTestID = 2;
+    UT_LOG("\n In %s [%02d%03d]\n", __FUNCTION__, gTestGroup, gTestID);
+
+    uint8_t numInputs = 0;
+    dsError_t ret   = dsERR_NONE;
+
+    UT_LOG_INFO("Calling dsHdmiInGetNumberOfInputs");
+    ret = dsHdmiInGetNumberOfInputs(&numInputs);
+    if(ret != dsERR_NONE)
+    {
+       UT_LOG_ERROR("Result: Failed to Get NumberOfInputsPorts:[%s]",ut_control_GetMapString(dsError_mapTable, ret));
+    }
+       
+    UT_LOG_INFO("Result: NumberOfInputsPorts:[%d]",numInputs);
+
+    if(!numInputs)
+       UT_LOG_INFO("Result: Platform does not supports HdmiIn Ports");
+
+    UT_LOG_INFO("Out %s", __FUNCTION__);
+}
+
+/**
+* @brief This test get number of input ports.
+*
+* This test function gets HdmiInput ports on platform.
+*
+* **Test Group ID:** 03@n
+* **Test Case ID:** 003@n
+*
+* **Test Procedure:**
+* Refer to Test specification documentation
+* [ds-hdmi-in_halSpec.md](../../docs/pages/ds-hdmi-in_halSpec.md)
+*/
+void test_l3_HdmiIn_get_status(void)
+{
+    gTestID = 3;
+    UT_LOG("\n In %s [%02d%03d]\n", __FUNCTION__, gTestGroup, gTestID);
+
+    dsError_t ret   = dsERR_NONE;
+    dsHdmiInStatus_t inputstatus;
+    
+    UT_LOG_INFO("Calling dsHdmiInGetStatus");
+    ret = dsHdmiInGetStatus(&inputstatus);
+    if(ret != dsERR_NONE)
+    {
+        UT_LOG_ERROR("Result: Failed to Get HDMI input port status:[%s]",ut_control_GetMapString(dsError_mapTable, ret));
+    }
+
+    UT_LOG_INFO("Result HDMI input port status isPresented: %s, activeport: %s",
+                 ut_control_GetMapString(bool_mapTable, inputstatus.isPresented),
+                 ut_control_GetMapString(dsHdmiInPort_mapTable, inputstatus.activePort));
+
+    for(int i = 0 ; i < dsHDMI_IN_PORT_MAX ; i++) {
+
+          if(!(inputstatus.isPresented))
+                 continue;
+
+           UT_LOG_INFO("Result: HDMI port connected info isPortConnected: %s, activeport: %s",
+                         ut_control_GetMapString(bool_mapTable, inputstatus.isPortConnected[i]),
```

Comment:
error code is not added?

Reply:
could not find , this line , think i have removed in not required test cases.

Reply:
line number 540. Error code is not added in the prints. check in all prints. Error codes has to be added